src/pybind/rados/setup.py

```diff
@@ -26,15 +26,30 @@
 __version__ = '2.0.0'
 
 
-def get_ceph_version():
+def get_version():
```

Comment:
`get_version()` was not called anywhere. the same applies to `get_ceph_version()`. the version number of the module is hardwired to `__version__` here.